5.6/Dockerfile.rhel7

```diff
@@ -0,0 +1,61 @@
+FROM rhel7
```

Comment:
We have a diff between this github repo and dist-git. We should probably discuss how we can minimize the diffs or automate them out somehow. If we no longer build RHEL images in jenkins then I'd say switch it to openshift3/sti-base.